CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 
 ## Improvements
 
-    ...
+- PR #111 Add and use unified build script
```

Comment:
Is this the correct description of this PR? Doesn't seem to add a new script. It just changes how the CUDA version is set.

Reply:
Not correct description. Correcting CHANGLOG.md to point to correct PR. Thanks for the review!